FEXCore/Source/Utils/MemberFunctionToPointer.h

```diff
@@ -17,24 +17,52 @@ class MemberFunctionToPointerCast final {
   public:
     MemberFunctionToPointerCast(PointerToMemberType Function) {
```

Comment:
This seems to lose the assertions. Is that intended?

Reply:
Intended, the asserts get moved to the accessor functions `GetConvertedPointer` and `GetVTableOffset` instead.